functionaltests/src/test/java/util/WebDriverPaydirekt.java

```diff
@@ -0,0 +1,59 @@
+package util;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * @author fhaertig
+ * @since 14.03.16
+ */
+public class WebDriverPaydirekt extends CustomWebDriver {
+
+    private static String LOGIN_FORM_USERNAME_FIELD_ID = "username";
+    private static String LOGIN_FORM_PASSWORD_FIELD_ID = "password";
+    private static String LOGIN_FORM_SUBMIT_BUTTON_NAME = "loginBtn";
+    private static String PAYMENT_SUBMIT_BUTTON_XPATH = "//BUTTON[@type='submit'][text()=' Jetzt bezahlen ']";
```

Comment:
Is it really the most reliable way to access the button?

Reply:
Thanks for extensive answers :) What other options did you try out?

Reply:
Now I locate the form, which I want to submit. Within this form, I locate the submit-button and then I click on it -> 0f6f0606